src/services/http.service.ts

```diff
@@ -61,12 +61,22 @@ class HttpService {
         continue;
       }
 
+      //eslint-disable-next-line
+      Object.entries(headers).map(([key, value]) => {
```

Comment:
I'm not sure it's the current place to add this logic. `buildUrl` meant (as the name suggests), to build the request url.
To manipulate headers, please add another method,